2024/2024-ssl-teams-vs-teams-passing-challenge.adoc

```diff
@@ -0,0 +1,105 @@
+* After *2 minutes of game time* a stop command will be issued and the first half of the game is over.
+* The half time break for this challenge will be only 60 seconds, since it should be mainly used to setup the robots at their desired position again and to exchange faulty robots. 
+* To start the second half a kick-off command for the blue team will be issued.
+* After another *2 minutes of game time* a stop command will be issued and the game is over. 
+
+
+=== Pass Definitions
+
+Teams are free in how they perform the passes. Definitions of a pass in general and additional constrains for valid pass that are considered in the scoring are given in this section.
+
+A pass is defined as follows:
+
+* A pass has a source position, a target position and a shooter.
+* The source position is where a robot kicks the ball.
+* The target position is where the ball stops or changes direction by more than *30 degrees* in field plane (x, y).
+* The shooter is the robot located nearest to the source position when the pass starts.
+* The pass distance is the euclidean distance between the source and target position.
+* The pass direction is the orientation of the `target - source` vector.
+* The pass is considered *chipped*, if the ball reached a maximum height of at least *15cm*.
+
+=== Special Ruleset during the technical challenge
+
+For the technical challenge a Div B field will be used. If the ball leaves the field via any of the field lines, the game will be halted and the team, that did not last touch the ball gets awared a free kick.
+
+During the games of the technical challenge only the following fouls from the ssl rulebook apply: 
+
+* Pushing 
+* Ball Holding 
+* Tipping over or dropping parts 
+* Boundary Crossing 
+* Excessive Dribbling 
+* Ball Speed 
+* Crashing 
+* Defender too close to ball 
+* Robot Stop Speed 
+
+During this challenge a maximum of 4 robots per team are allowed. For every yellow card, that a team got, the number of allowed robots for the half time will be decreased by one. A red card will decrease that number for the entire time of the game. 
+
+== Scoring
+
+A valid pass must meet the following criteria:
+
+* The pass distance must be at least [.line-through]#*1.5m*.# *1 m*.
+* [.line-through]#The shooter of the previous pass can not be the receiver of the current pass.#
+* The shooter and the reciever of the pass must be from the same team.
+* The pass direction of the pass must differ by at least *10 degrees* from the inverted previous pass direction: latexmath:[abs(passDirection_{cur} - (passDirection_{previous} + 180°)) > 10°]
+* The ball must not exceed 6.5m/s (as defined in the tournament rules).
+* The source and target must be inside field lines.
+
+NOTE: Previous passes include all passes, not only those considered valid.
+
+Passes are scored as follows:
+
+* A valid ground pass -- *1 point*
+* A valid chipped pass -- [.line-through]#*5 points*# *3 points*
```

Comment:
I tend to give only one point for chip as well for two reasons:
1. A chip kick can already be an advantage, as it is harder to intercept
2. The focus of this challenge is different than last year: It is not about being able to chip pass, but to avoid passes being intercepted. So I would not account for the way the pass was played.

Reply:
Sounds like a good it, since it will also make judging more easy and no possible discussions, whether the 15 cm wear exceeded or not.
In the next commit I also added the removal of all the corresponding rules as suggestion.